### PR TITLE
1.  reformat code + run code cleanup

### PR DESCRIPTION
**Pull request by Andy2003**

### CODE REVIEW

File: src/main/kotlin/org/neo4j/graphql/CrudOperations.kt [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
     val typeName = type.name
     val idField = type.fieldDefinitions.find { it.type.name() == "ID" }
     val scalarFields = type.fieldDefinitions.filter { it.type.isScalar() }.sortedByDescending { it == idField }
-    val has_idField = type.fieldDefinitions.find { it.name.equals("_id") } != null
+    val hasIdField = type.fieldDefinitions.find { it.name == "_id" } != null
     val idFieldArg = idField?.let { it.name + ":" + it.type.render() }
 
     val result = if (ctx.mutation.enabled && !ctx.mutation.exclude.contains(typeName) && scalarFields.isNotEmpty()) {
-        val fieldArgs = scalarFields.map { it.name + ":" + it.type.render() }.joinToString(", ")
+        val fieldArgs = scalarFields.joinToString(", ") { it.name + ":" + it.type.render() }
```
Comment (jexp) [inline src/main/kotlin/org/neo4j/graphql/CrudOperations.kt]: I don't like this style at all it makes it harder to understand what's happening. As joining comes after the mapping not before.

File: src/main/kotlin/org/neo4j/graphql/Predicates.kt [hunk 2]
```diff
@@ -88,98 +89,99 @@ fun arrows(outgoing: Boolean?): Pair<String, String> {
 }
 
 
-data class RelationshipInfo(val objectType:GraphQLObjectType, val directive:Directive, val type:String, val out:Boolean?, val startField: String? = null,val endField: String? = null, val isRelFromType: Boolean = false) {
+data class RelationshipInfo(val objectType: GraphQLObjectType, val directive: Directive, val type: String, val out: Boolean?, val startField: String? = null, val endField: String? = null, val isRelFromType: Boolean = false) {
     val arrows = arrows(out)
-    val label = objectType.name
+    val label: String = objectType.name
```
Comment (jexp) [inline src/main/kotlin/org/neo4j/graphql/Predicates.kt]: don't need the type info

File: src/main/kotlin/org/neo4j/graphql/Translator.kt [hunk 3]
```diff
@@ -181,36 +186,35 @@ class Translator(val schema: GraphQLSchema) {
...
     private fun orderBy(variable: String, args: MutableList<Argument>): String {
-        val arg = args.find { it.name == "orderBy"}
-        val values = arg?.value?.let { when (it) {
-            is ArrayValue -> it.values.map { it.toJavaValue().toString() }
-            is EnumValue -> listOf(it.name)
-            is StringValue -> listOf(it.value)
-            else -> null
-        }}
+        val arg = args.find { it.name == "orderBy" }
+        val values = arg?.value?.let { it ->
```
Comment (jexp) [inline src/main/kotlin/org/neo4j/graphql/Translator.kt]: don't need `it ->`

File: src/main/kotlin/org/neo4j/graphql/Translator.kt [hunk 4]
```diff
@@ -554,18 +550,19 @@ class Translator(val schema: GraphQLSchema) {
 
 
 object SchemaBuilder {
```
Comment (jexp) [inline src/main/kotlin/org/neo4j/graphql/Translator.kt]: we should probably pull it out of the translator file

File: src/main/kotlin/org/neo4j/graphql/Translator.kt [hunk 5]
```diff
@@ -605,33 +604,39 @@ object SchemaBuilder {
...
         }
-        augmentations.flatMap {listOf(it.create, it.update, it.delete, it.merge)
-                .filter { it.isNotBlank() && mutationDefinition.fieldDefinitions.none { fd -> it.startsWith(fd.name + "(") } }}
-                .let {
-                    if (!it.isEmpty()) {
-                        val newQueries = schemaParser.parse("type AugmentedMutation { ${it.joinToString("\n")} }").getType("AugmentedMutation").get() as ObjectTypeDefinition
-                        typeDefinitionRegistry.remove(mutationDefinition)
-                        typeDefinitionRegistry.add(mutationDefinition.transform { mdb -> newQueries.fieldDefinitions.forEach { mdb.fieldDefinition(it) } })
-                    }
+        augmentations.flatMap { it ->
```
Comment (jexp) [inline src/main/kotlin/org/neo4j/graphql/Translator.kt]: not `it ->`

File: src/main/kotlin/org/neo4j/graphql/Predicates.kt (hunk 2 above)
Comment (Andy2003) [reply src/main/kotlin/org/neo4j/graphql/Predicates.kt]: IntelliJ annotes this with: Declaration has type inferred from a platform call, which can lead to unchecked nullability issues. Specify type explicitly as nullable or non-nullable.

File: src/main/kotlin/org/neo4j/graphql/Translator.kt (hunk 4 above)
Comment (jexp) [reply src/main/kotlin/org/neo4j/graphql/Translator.kt]: See PR #50 coolio

Comment (Andy2003) [reply src/main/kotlin/org/neo4j/graphql/Translator.kt]: done in the next PR

**Pull request merged**